src/ConsentRequiredController.js

```diff
@@ -24,6 +25,10 @@ export default FormController.extend({
     this.model.set('expiresAt', this.options.appState.get('expiresAt'));
     this.model.set('scopes', this.options.appState.get('scopes'));
     this.listenTo(this.form, 'cancel', _.bind(this.model.cancel, this.model));
+
+    // add Skip to main content link
+    const skipLink = new SkipLink();
+    $('body').find('#okta-login-container').prepend(skipLink.render().$el);
```

Comment:
```
$(#${Enums.WIDGET_CONTAINER_ID}).find(..).prepend(..)
```

Reply:
Ok, what's the rule for when to add a class/id to the Enums file?

Reply:
I added a new one: https://github.com/okta/okta-signin-widget/pull/1540/commits/e50036e8415fd0c72e42204680f3fbcf4f4deaff#diff-6cd1196e6f18f0fef5f0cb85e1d32142bfa44deacee6ba54a3d26d1521b06068R16